### PR TITLE
Document `apso` process

### PR DESCRIPTION
**Pull request by xRuiAlves**
Adding some needed documentation on various processes within `apso`, namely on how to release new versions.

### CODE REVIEW

File: PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+It's recommended to open a PR with the Changelog changes so that they can be reviewed by someone else from the team.
+
+### Publishing a snapshot version
+
+To publish a snapshot version to [Nexus Sonatype](https://oss.sonatype.org), simply use `sbt` to run `publish`. The 
+version will be published under the [Velocidi Snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/com/velocidi/). 
+
+### Releasing artifacts
+
+To release the artifacts in the [Velocidi Maven Repository](https://repo1.maven.org/maven2/com/velocidi/), simply use
```
Comment (jcazevedo) [inline PROCESS.md:36]: Nitpick: We don't release artifacts directly to Maven Central, but rather to Sonatype's releases repository, which eventually gets synced to Maven Central.

File: PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Process
+
+This document describes the practices we use when developing and maintaining `apso`.
```
Comment (jcazevedo) [inline PROCESS.md:3]: I don't think we need to mention Apso in a monospaced font.

```suggestion
This document describes the practices we use when developing and maintaining Apso.
```

**Review by jcazevedo — CHANGES_REQUESTED** [review]
Thanks for this!

File: PROCESS.md (hunk 1 above)
Comment (xRuiAlves) [reply PROCESS.md]: Fixed! 👍

File: PROCESS.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Before relasing, create, if it doesn't exist yet, a [Changelog](CHANGELOG.md) entry for the version you're releasing, 
+following the template.
+
+It's recommended to open a PR with the Changelog changes so that they can be reviewed by someone else from the team.
+
+### Publishing a snapshot version
+
+To publish a snapshot version to [Nexus Sonatype](https://oss.sonatype.org), simply use `sbt` to run `publish`. The 
+version will be published under the [Velocidi Snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/com/velocidi/).
```
[the remark this answers is not in the document]
Comment (xRuiAlves) [reply PROCESS.md]: Fixed! 👍

File: PROCESS.md (hunk 2 above)
Comment (xRuiAlves) [reply PROCESS.md]: Updated! 👍

File: PROCESS.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+### Updating the Changelog
+
+Before relasing, create, if it doesn't exist yet, a [Changelog](CHANGELOG.md) entry for the version you're releasing, 
+following the template.
+
+It's recommended to open a PR with the Changelog changes so that they can be reviewed by someone else from the team.
+
+### Publishing a snapshot version
+
+To publish a snapshot version to [Nexus Sonatype](https://oss.sonatype.org), simply use `sbt` to run `publish`. The
```
[the remark this answers is not in the document]
Comment (xRuiAlves) [reply PROCESS.md]: Fixed! 👍

File: PROCESS.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+### Publishing a snapshot version
+
+To publish a snapshot version to [Nexus Sonatype](https://oss.sonatype.org), simply use `sbt` to run `+publish`. The 
+version will be published in [Sonatype's snapshots repository, under the Velocidi organization](https://oss.sonatype.org/content/repositories/snapshots/com/velocidi/). 
+
+### Releasing artifacts
+
+To release the artifacts in the Sonatype's release repository, which eventually gets synced to [Maven Central](https://repo1.maven.org/maven2/com/velocidi/)
+, simply use `sbt` to run `release`.
```
Comment (jcazevedo) [inline PROCESS.md:37]: Nitpick: This `,` in a newline gets rendered with a space before. Can we move it to the line above?

**Review by jcazevedo — CHANGES_REQUESTED** [review]

File: PROCESS.md (hunk 5 above)
Comment (xRuiAlves) [reply PROCESS.md:37]: Oops, my bad

Comment (xRuiAlves) [reply PROCESS.md]: Should be alright now! 👍

**Review by jcazevedo — APPROVED** [review]
Thanks a lot for this!

**Pull request merged**